geniza/corpus/models.py

```diff
@@ -468,6 +468,12 @@ def index_data(self):
         # script+language when/if included in index data
     }
 
+    def save(self, *args, **kwargs):
+        # Auto-increment pk when none is provided
+        if self.id is None:
+            self.id = self.__class__.objects.all().order_by("-id")[0].id + 1
+        super(self.__class__, self).save(*args, **kwargs)
```

Comment:
Was there a more straightforward way to doing this?

Reply:
@kmcelwee django should do this automatically for us if no id is specified, we already have it set as an auto-increment. Did that not work?

Reply:
I removed this — I had changed two things at once, not realizing that casting an empty string as `None` was sufficient. Thanks!